src/proxy/processors/push-action/parsePush.ts

```diff
@@ -111,6 +112,9 @@ const getCommitData = (contents: CommitContent[]) => {
       const authorEmail = author?.split(' ').reverse()[2].slice(1, -1);
       console.log({ authorEmail });
 
+      const committerEmail = committer?.split(' ').reverse()[2].slice(1, -1);
```

Comment:
since committer has already been split and reversed on line 109 can this be reused please. And grouping the committer stuff is easier to read.

```js
const committerArr = committer?.split(' ').reverse()
const commitTimestamp = committerArr[1]
// slice to trim start and end from `<an-email@address.whatever>`
const committerEmail = committerArr[2].slice(1, -1)
```

Also a comment for why the slice is relevant. I'm assuming it's for `<>` or something else wrapping the email so I've proposed one.

Reply:
Good call, theres some other similar rationalisation that can be applied to that function (cleanup of `author` and `committer`, grouping the operations better etc.) which I've also applied.

I've reduce some of the (excessive and duplicative IMHO) log messaging to the debug level. TBH I think we could remove all but the first and last log message (input data and output data). WDYT @06kellyjac ?

Reply:
removed the debug level logs!